x-pack/plugins/ingest_manager/server/services/epm/fields/field.ts

```diff
@@ -27,6 +27,7 @@ export interface Field {
   ignore_above?: number;
   object_type?: string;
   scaling_factor?: number;
+  dynamic?: 'strict' | boolean;
```

Comment:
@skh For my knowledge: What does `strict` mean here?

Reply:
`false` ignores new fields and the document is indexed without them, while `strict` throws an error when they appear and the document isn't indexed at all. (See https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic.html )

Reply:
🤦 I was thinking it means something in typescript. Makes sense now.

Reply:
Ah :) Yes, you can use a string literal instead of a type in TypeScript. It's less useful as it looks because it doesn't give us run time validation, but at least it's sort-of self-documenting code that way.